pkgs/applications/audio/noisetorch/default.nix

```
@@ -0,0 +1,48 @@
+{ stdenv, cmake, buildGoModule, fetchgit }:
+
+buildGoModule rec {
+  name = "NoiseTorch-${version}";
+  version = "0.5.1-beta";
+  rev = "37b35ce1eed438adee42e50d0cc0ddf48375846e";
+
+
+  patches = [ ./version.patch ];
+
+  src =  fetchgit {
+    inherit rev;
+    url = "https://github.com/lawl/NoiseTorch";
+    sha256 = "076sfbb3p5zligz0r9rdzy2bhcl2agjvwpi8iw9p118j56ni4941";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  subPackages = [ "." ];
+
+ # modSha256 = "1f585ql8qj8jryrv4bx7hdisssfspk3axx5qp0crfd57p893w5qh";
+
+  vendorSha256 = "1f585ql8qj8jryrv4bx7hdisssfspk3axx5qp0crfd57p893w5qh";
+
+  preBuild = ''
+    pushd ./librnnoise_ladspa  
+    cmake  . -DBUILD_VST_PLUGIN=OFF -DBUILD_LV2_PLUGIN=OFF -DBUILD_LADSPA_PLUGIN=O 
+    make
+    popd
+    go generate;
+    rm  ./scripts/*
+  '';
+
+  postInstall = ''
+     mkdir -p $out/share/icons/hicolor/256x256/apps/
+     cp assets/icon/noisetorch.png $out/share/icons/hicolor/256x256/apps/
+     mkdir -p $out/share/applications/
+     cp assets/noisetorch.desktop $out/share/applications/
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Virtual microphone device with noise supression for PulseAudio";
+    homepage = https://github.com/lawl/NoiseTorch;
```

```suggestion
    homepage = "https://github.com/lawl/NoiseTorch";
```
Unquoted URLs are deprecated.